tests/generate_serialized_jsons.php

```diff
@@ -150,7 +150,9 @@
             ->setOrigin($settings['origin'])
             ->setGatherMethods($gatherMethod)
             ->setPrefix('Api')
-            ->setDestination(TestCase::getTestDirectory());
+            ->setDestination(TestCase::getTestDirectory())
+            ->setSchemasSave(false)
+            ->setSchemasFolder('wsdl');
```

Comment:
Did you run this script with command line? This should update the tests/generated/parsed_*.json files with the new options.